app/Importer/LocationImporter.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace App\Importer;
+
+use App\Models\Location;
+
+/**
+ * When we are importing users via an Asset/etc import, we use createOrFetchUser() in
+ * App\Importer.php. [ALG]
+ *
+ * Class UserImporter
```

Comment:
The comment above should be updated too but I'm not sure to what.

```suggestion
 * Class LocationImporter
```